core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -445,13 +449,13 @@ contract Liquidatable is PricelessPositionManager {
     }
 
     /**
-     * @dev This overrides pfc() so the Liquidatable contract can report its profit from corruption.
+     * @dev This returns the profit from corruption from active and liquidated positions.
      */
-    function pfc() public override view returns (FixedPoint.Unsigned memory) {
-        return super.pfc().add(_getFeeAdjustedCollateral(rawLiquidationCollateral));
+    function pfc() public override view nonReentrantView() returns (FixedPoint.Unsigned memory) {
```

Comment:
nit: why not just implement this in `feePayer()`? Then all the derived contracts would have to implement is `_pfc()`? Just feels a little weird implementing this here since there's nothing barring the base class from just implementing it once.

Reply:
done good point